CMakeLists.txt

```diff
@@ -851,6 +851,8 @@ if (LLVM_ENABLE_DOXYGEN)
   message(STATUS "Doxygen: enabled")
 endif()
 
+check_symbol_exists(getauxval "sys/auxv.h" HAVE_GETAUXVAL)
```

Comment:
I think that it would be nicer to move this check into the runtime CMakeLists.txt.  This will ensure that we can do standalone builds in the future for the runtime as well as home it along with the site where it will be used.

Reply:
Done.
But global `include/swift/Config.h.in` is modified based on this variable, not special runtime header.